stdlib/runtime/numbers.gr

```diff
@@ -2465,11 +2030,22 @@ provide let coerceNumberToFloat64 = (x: Number) => {
   WasmI32.toGrain(result): Float64
 }
 
+@unsafe
+let _SIMPLE_NUMBER_MAX = 0x3FFFFFFFn
+@unsafe
+let _SIMPLE_NUMBER_MIN = 0xC0000001n
```

Comment:
Since you added these, can you move them to the top and use them everywhere we do shifts on `I32_MAX` and `I32_MIN`?

Reply:
Oh, these are tagged, aren't they.

Reply:
No these are just the values; they should be equivalent to shifting I32_MAX